src/platform/datapath_raw.c

```diff
@@ -610,11 +610,12 @@ CXPLAT_THREAD_CALLBACK(CxPlatRouteResolutionWorkerThread, Context)
                     Operation->Callback(
                         Operation->Context, Operation->IpnetRow.PhysicalAddress, Operation->PathId, TRUE);
                 }
-                CxPlatPoolFree(&Worker->OperationPool ,Operation);
             } else {
                 Operation->Callback(
                     Operation->Context, Operation->IpnetRow.PhysicalAddress, Operation->PathId, TRUE);
             }
+
+            CxPlatPoolFree(&Worker->OperationPool ,Operation);
```

Comment:
This is a bugfix, right? It isn't mentioned in the PR description.

Reply:
To add on, formatting is incorrect (Comma is in the wrong spot).

Reply:
yes